orbs/changelog/Dockerfile.publish

```diff
@@ -13,10 +15,21 @@ RUN apt-get update -y && \
       # checkout
       git openssh-client \
       # node installation
-      bash curl \
+      bash curl wget \
```

Comment:
__rmk__

As you use `wget`, perhaps you could remove `curl`

Reply:
Good catch 👀 !